src/Geodesy.jl

```diff
@@ -1,32 +1,53 @@
 module Geodesy
 
+using CoordinateTransformations
+using FixedSizeArrays
 using Compat
 
+import CoordinateTransformations.transform,
+       CoordinateTransformations.transform,
+       CoordinateTransformations.transform_deriv_params,
+       CoordinateTransformations.compose,
+       CoordinateTransformations.∘
```

Comment:
Sure, but I had to also export them below, so I had the list (helps me remember)